src/heart.c

```diff
@@ -489,6 +491,8 @@ do_terminate(int reason)
     case R_CLOSED:
     case R_ERROR:
     default:
+        /* sleep for a short period of time to allow --run-on-exit script in erlinit.config to run */
+        sleep(WAIT_TO_KILL);
         kill_old_erlang(reason);
         reboot(LINUX_REBOOT_CMD_RESTART);
```

Comment:
no need for the reboot command in the /sbin/exit.sh

Reply:
Not every case in the case statement here causes a reboot, so it is still needed.